tox.ini

```diff
@@ -35,9 +8,10 @@ norecursedirs = .* docs requirements
 
 [testenv]
 deps =
-    django18: Django>=1.8,<1.9
     django111: Django>=1.11,<2.0
     django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2
     -r{toxinidir}/requirements/test.txt
 commands =
     py.test {posargs}
```

Comment:
Need `python -Wd -m pytest {posargs}` to enable display of any applicable deprecation warnings.